guvnor-ala/guvnor-ala-openshift/guvnor-ala-openshift-ui/guvnor-ala-openshift-ui-client/src/main/java/org/guvnor/ala/ui/openshift/client/resources/i18n/GuvnorAlaOpenShiftUIConstants.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.ala.ui.openshift.client.resources.i18n;
+
+public class GuvnorAlaOpenShiftUIConstants {
```

Comment:
Similar to above, It's a "place holder" for the upcoming OpenShift provider UI constants.
We don't reached the point of needing constants yet, but we will soon for example for the error messages at the fields, like "please enter the openshift url" or whatever.
I'd suggest to let it be.